cpp/daal/src/algorithms/linear_model/oneapi/linear_model_predict_dense_default_batch_oneapi_impl.i

```diff
@@ -66,8 +67,10 @@ services::Status PredictKernelOneAPI<algorithmFPType, defaultDense>::addBetaInte
     KernelArguments args(4, status);
     DAAL_CHECK_STATUS_VAR(status);
     args.set(0, betaTable, AccessModeIds::read);
+    DAAL_ASSERT(nBetas <= services::internal::MaxVal<uint32_t>::get());
     args.set(1, nBetas);
     args.set(2, yTable, AccessModeIds::write);
+    DAAL_ASSERT(yNCols <= services::internal::MaxVal<uint32_t>::get());
     args.set(3, yNCols);
```

Comment:
Is it uint in the kernel? Please add static cast if implicit conversion takes place.